java/test/src/main/java/io/ray/test/PlacementGroupTest.java

```diff
@@ -37,7 +40,10 @@ public void testCreateAndCallActor() {
 
     // Test creating an actor from a constructor.
     ActorHandle<Counter> actor =
-        Ray.actor(Counter::new, 1).setPlacementGroup(placementGroup, 0).remote();
+        Ray.actor(Counter::new, 1)
+            .setResource("CPU", 1.0)
+            .setPlacementGroup(placementGroup, 0)
```

Comment:
Will it be more clear to call like `setPlacementGroup(placementGroup).setPlacementGroupIndex(0)`?

Reply:
That's maybe a better design, but actually, I don't think we should involve this core API change in this pr